activerecord/test/cases/adapters/mysql2/arel_table_alias_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+require "support/connection_helper"
+require "models/aircraft"
+
+module ActiveRecord
+  class MysqlArelTableAliasTest < ActiveRecord::Mysql2TestCase
+    test "it should create record with custom arel_table.table_alias" do
+      Aircraft.arel_table.table_alias = "a"
+      Aircraft.create!
+    end
+
+    test "it should update record with custom arel_table.table_alias" do
+      Aircraft.arel_table.table_alias = "a"
+      aircraft = Aircraft.create!
+      aircraft.name = "foo"
+      aircraft.save!
+    end
+
+    test "it should delete record with custom arel_table.table_alias" do
+      Aircraft.arel_table.table_alias = "a"
+      aircraft = Aircraft.create!
+      e = assert_raises(ActiveRecord::StatementInvalid) do
+        aircraft.delete
+      end
+      p e.sql
```

Comment:
```suggestion
      aircraft.delete
```

Reply:
This was a debug commit I believe I already removed this however currently there are debugger statements in the commit that I will remove later tonight when I'm done debugging MySQL 5.6 and Mariadb.